traefik/Changelog.md

```diff
@@ -1,31 +1,130 @@
 # Change Log
 
-## 27.0.0 (unreleased)
+## 27.0.0  ![AppVersion: v2.11.0](https://img.shields.io/static/v1?label=AppVersion&message=v2.11.0&color=success&logo=) ![Kubernetes: >=1.16.0-0](https://img.shields.io/static/v1?label=Kubernetes&message=%3E%3D1.16.0-0&color=informational&logo=kubernetes) ![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
 
-**Upgrade notes**
```

Comment:
We should probably restore this section manually now after the notes have been auto-generated!

Reply:
Fixed with my last update